modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -79,13 +94,70 @@ const viewportUpdate = WindowVisualViewport.get().fold(
   }
 );
 
-const toggleFullscreen = (editor: Editor, fullscreenState: Cell<any>) => {
+const getFullscreenRoot = (editor: Editor): SugarElement<Element> => {
```

Comment:
Could you move this fullscreen logic to its own module please (eg `NativeFullscreen.ts` or similar)? It'll just make it a little cleaner.

Reply:
Sure

Reply:
Thanks!